### PR TITLE
feat(example): Error Page

### LINKED ISSUE #757: [EXAMPLE] Error page
Create Error Page example
For mobile https://www.figma.com/file/hrwE3VsMBHgdJoS86rVr4W/Desktop-and-Mobile-Vue-Storefront?node-id=551%3A93

### PR DESCRIPTION
**Pull request by martaradziszewska**
# Related issue
Closes #757 

# Scope of work
Created Error page example

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed

If applicable:

- [x] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

**Comment by aniamusial** [comment]
You can move icons svgs to icons.js

**Comment by martaradziszewska** [comment]
@pspaczek updated :)

File: packages/vue/public/assets/storybook/Error/error.svg [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<svg width="144" height="144" viewBox="0 0 124 117" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (pspaczek) [inline packages/vue/public/assets/storybook/Error/error.svg]: @martaradziszewska why we have 2 `svg` files? Probably move it to shared is will be a good idea, what do you think?

File: packages/vue/src/examples/pages/error/Error.vue [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<template>
+  <div id="error">
+    <SfImage
+      class="image"
+      :src="{
```
Comment (pspaczek) [inline packages/vue/src/examples/pages/error/Error.vue]:
```suggestion
      :src="require('@storefront-ui/shared/..')
```
for .svg we dont need mobile / desktop files

File: packages/vue/src/examples/pages/error/Error.vue [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+#error {
+  box-sizing: border-box;
+  padding: 0 1.25rem;
+  @include for-desktop {
+    max-width: 1240px;
+    margin: auto;
+  }
+}
+.image {
+  padding: var(--spacer-extra-big);
```
Comment (pspaczek) [inline packages/vue/src/examples/pages/error/Error.vue]:
```suggestion
  margin: 0 0 var(--spacer-extra-big) 0;
```

File: packages/vue/src/examples/pages/error/Error.vue [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  components: { SfButton, SfImage, SfHeading }
+};
+</script>
+
+<style lang="scss" scoped>
+@import "~@storefront-ui/vue/styles";
+
+#error {
+  box-sizing: border-box;
+  padding: 0 1.25rem;
```
Comment (pspaczek) [inline packages/vue/src/examples/pages/error/Error.vue]:
```suggestion
  padding: 0 var(--spacer-big);
```

Comment (pspaczek) [reply packages/vue/src/examples/pages/error/Error.vue]: can we use height:100% and center content on desktop and space-between on mobile?
image

**Comment by pspaczek** [comment]
thanks @martaradziszewska

**Pull request merged**